cloud-stream-manual-ack/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.solace.samples.spring.scs</groupId>
+	<artifactId>cloud-stream-manual-ack</artifactId>
+	<version>0.0.2-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>cloud-stream-manual-ack</name>
+	<description>Sample Manual Ackonknowledgemnt for Spring Cloud Stream using Solace PubSub+ binder</description>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.4</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+
+	<properties>
+		<spring-cloud.version>2021.0.1</spring-cloud.version>
+		<solace-spring-cloud-bom.version>2.3.0</solace-spring-cloud-bom.version>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>${spring-cloud.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>com.solace.spring.cloud</groupId>
+				<artifactId>solace-spring-cloud-bom</artifactId>
+				<version>${solace-spring-cloud-bom.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.solace.spring.cloud</groupId>
+			<artifactId>spring-cloud-starter-stream-solace</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.solace.samples.spring</groupId>
+			<artifactId>spring-samples-datamodel</artifactId>
+			<version>0.0.1-SNAPSHOT</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
```

Comment:
I believe we can remove this requirement here. This will allow us to remove an extra dependency + remove the `server.port` parameter in the config.